framework/validators/UniqueValidator.php

```diff
@@ -164,13 +165,39 @@ public function validateAttribute($model, $attribute)
             }
         }
 
-        if ($exists) {
-            if (count($targetAttribute) > 1) {
-                $this->addComboNotUniqueError($model, $attribute);
-            } else {
-                $this->addError($model, $attribute, $this->message);
+        return $exists;
+    }
+
+    /**
+     * Apply filters and prepare a query
```

Comment:
Other way around to be more consistent with the method naming: "Prepares a query by applying filters"